src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml

```diff
@@ -203,7 +198,7 @@
                         </li>
                     </ol>
                     <form asp-controller="Consent" asp-action="Index" asp-route-returnUrl="@ViewData["ReturnUrl"]" method="post">
-                        <input type="hidden" asp-for="ReturnUrl" />
+                        <input type="hidden" asp-for="ReturnUrl"/>
```

Comment:
I find self-closing tags are more readable with the space before the `/>` (as they were).

Reply:
👍